apps/src/lib/kits/maker/ui/SetupInstructions.jsx

```diff
@@ -38,12 +37,30 @@ const style = {
 };
 
 export default class SetupInstructions extends React.Component {
```

Comment:
Good call! Especially since some of these components have internal logic, it definitely seems worth adding a follow-up where we break these out into individual files.